packages/message-db/message-db/test/message-db.test.ts

```diff
@@ -56,10 +46,7 @@ namespace CartService {
   }
 
   export function createWithSnapshotStrategyAndCaching(context: MessageDbContext) {
-    const access = AccessStrategy.AdjacentSnapshots<E, S>(
-      Cart.Fold.snapshotEventType,
-      Cart.Fold.snapshot,
-    )
+    const access = AccessStrategy.AdjacentSnapshots<E, S>(Cart.Fold.snapshotEventType, Cart.Fold.snapshot)
```

Comment:
maybe a common helper that has L50+L51 so the fact that it's all the same except for `access` stands out?